scripts/complexity-analysis.js

```diff
@@ -4,7 +4,9 @@ const plato = require('es6-plato');
 // parse command line args
 const optionDefinitions = [
     { name: 'src', type: String, defaultValue: './es6-src/**' },
-    { name: 'output', type: String, defaultValue: './report' }
+    { name: 'output', type: String, defaultValue: './report' },
+    { name: 'version', type: String, defaultValue: '2.0.0' },
```

Comment:
What is this `2.0.0` here? Is that VoTT version?  If so can we pull out of package.json like you do above?

Reply:
Or is the the value only if version is not already supplied?

Reply:
just a default value, if one isn't passed in.